packages/stickers/README.md

```diff
@@ -0,0 +1,3 @@
+# Stickers
+
+SVG stickers.
```

Comment:
💭 Should we add more description here? I think it would be helpful to have a description of what a sticker is here, and maybe even a link back to the docs describing how to use it.

Reply:
Added some basic docs. https://github.com/google/web-stories-wp/pull/7265/commits/da0d17c2bb06ecb6309f680e5be0c630dbb71821